core/postsolve.gms

```diff
@@ -225,9 +225,9 @@ if(cm_iterative_target_adj eq 7,
     p_taxCO2eq_until2150(t,regi) = pm_taxCO2eq(t,regi);
 	); 
  
-p_actualbudgetco2(t) =           sum(ttot$(ttot.val < t.val AND ttot.val > 2010), (sum(regi, (vm_emiTe.l(ttot,regi,"co2") + vm_emiCdr.l(ttot,regi,"co2") + vm_emiMac.l(ttot,regi,"co2"))) * sm_c_2_co2 * pm_ts(ttot)))
```

Comment:
Does it make sense to have p_actualbudgetco2 for t < 2020?

Reply:
no, it doesn't. your changes look good to me